packages/pds/src/event-stream/background-queue.ts

```diff
@@ -5,10 +5,13 @@ import { dbLogger } from '../logger'
 // A simple queue for in-process, out-of-band/backgrounded work
 
 export class BackgroundQueue {
-  queue = new PQueue()
+  queue = new PQueue({ concurrency: 10 })
   constructor(public db: Database) {}
 
   add(task: Task) {
+    if (this.queue.isPaused) {
```

Comment:
wait actually! can we log here?

Reply:
just so we know when something is skipped during shutdown

Reply:
Will add this in soon with task names 👍